src/react-image-zooom.js

```diff
@@ -202,14 +202,14 @@ function ImageZoom({
         onTouchMove={handleMove}
         onTouchEnd={handleLeave}
       >
-        <Img
+        {imgData && <Img
           id="imageZoom"
           src={imgData}
```

Comment:
why not:
`src={imgData ? imgData : ''}`

so that the image tag is added to the DOM and avoid layout shifts when the useEffect runs

Reply:
@Mario-Duarte according to [HTML spec](https://html.spec.whatwg.org/#the-img-element) it's forbidden. Here is quote: "The [src](https://html.spec.whatwg.org/#attr-img-src) attribute must be present, and must contain a [valid non-empty URL potentially surrounded by spaces](https://html.spec.whatwg.org/#valid-non-empty-url-potentially-surrounded-by-spaces) referencing a non-interactive, optionally animated, image resource that is neither paged nor scripted."

In my opinion it's better to follow to official spec to avoid problems in the future. 
About shifts good point. Maybe we can add image's height and width to his parent for avoiding layout shifts?